lib/telemetry.ex

```diff
@@ -29,13 +33,15 @@ defmodule Logflare.Telemetry do
 
   @metrics_interval 30_000
 
+  @user_specific_tags [:source_id, :source_token, :backend_id]
+
   @impl true
   def init(_arg) do
     otel_exporter =
       if Application.get_env(:logflare, :opentelemetry_enabled?) do
         otel_exporter_opts =
           Application.get_all_env(:opentelemetry_exporter)
-          |> Keyword.put(:metrics, metrics())
+          |> Keyword.put(:metrics, metrics() |> add_filters())
```

Comment:
not a bad idea, but we want to keep the subset of metrics exposed to users small and intentional for now. a separate function for user-specific metrics would be better.

we only want one specific metric for a start:
- count of events ingested by a backend for a source (both source and backend must match)